samples/boards/altera_max10/sysid/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include "altera_avalon_sysid.h"
+
+void main(void)
+{
+	s32_t sysid;
+
+	sysid = alt_avalon_sysid_test();
+	if (!sysid) {
+		printk("[SysID] hardware and software appear to be in sync\n");
```

Comment:
If this is the expected output, can we just turn this into a test case and not a sample, where success/failure is reported via the typical test case way?
We could then remove the build_only tag.

Reply:
do you want me to move this to ztest framework?

Reply:
what I want is for this to be tested at runtime, and report pass/fail like other test cases, and get rid of build_only
This should be a test, not a sample

Reply:
put it in tests/boards/....